packages/api-gw/src/adapters/link/link.integration.js

```diff
@@ -0,0 +1,100 @@
+const fastify = require('fastify');
+const linkAdapters = require('./link');
+const Faker = require('faker');
+const linkRouterSchema = require('../../delivery/http/routes/link/schema');
+
+describe('Link Adapters', () => {
+  let server;
+
+  beforeEach(async () => {
+    server = fastify({});
+  });
+
+  afterEach(async function () {
+    if (server) {
+      await server.close();
+      server = null;
```

Comment:
@omarefg can we remove this line? because it is not longer needed due to the `afterEach` and `describe` statements already makes a clean-up of the dangling references.
```suggestion
```